lib/jsonapi/plugs/query_parser.ex

```diff
@@ -182,7 +182,7 @@ defmodule JSONAPI.QueryParser do
       if inc =~ ~r/\w+\.\w+/ do
         acc ++ handle_nested_include(inc, valid_includes, config)
       else
-        inc = inc |> String.to_existing_atom()
+        inc = String.to_atom(inc)
```

Comment:
Are these changes a possible DOS vector?

Reply:
Yea, it might be easier to try/catch around this?

Reply:
Done!